igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitlabCiProperties.java

```diff
@@ -50,9 +50,9 @@ public void setMasters(List<GitlabCiHost> masters) {
     @NotEmpty private String name;
     @NotEmpty private String address;
     private String privateToken;
-    private boolean limitByMembership = false;
-    private boolean limitByOwnership = true;
-    private Integer itemUpperThreshold;
+    private boolean limitByMembership = true;
```

Comment:
Changing these defaults could also break folks.  I'd also love to know the reason for changing them.

Reply:
@dbyron-sf I don't think this code was ever fully working or finished, it doesn't work without this PR in Echo: https://github.com/spinnaker/echo/pull/1148/files and there are several closed issues related to adding Gitlab CI support that went stale.  

The default change I feel make sense though.  The defaults before required the Gitlab account that created the API token to be an owner in any project for it to appear in the API request to Gitlab.  The new defaults only require it is a member with appropriate read access.   Personally I wouldn't want to give owner access to an account just to trigger Spinnaker (that would give it full privileges to the entire project and all Gitlab features).

Reply:
Fair enough...especially if you'll be around for the next little while to help support folks who pick up this change :)